test/membrane_webrtc/integration_test.exs

```diff
@@ -358,6 +379,11 @@ defmodule Membrane.WebRTC.IntegrationTest do
         webrtc: %WebRTC.Source{signaling: {:whip, ip: {127, 0, 0, 1}, port: 6789}}
       )
 
+      [send_pipeline, receive_pipeline]
+      |> Enum.each(fn pipeline ->
+        assert_pipeline_notified(pipeline, :webrtc, {:negotiated_video_codecs, [:vp8]})
```

Comment:
All the added assertions only test one case, I don't think it helps much. I meant to add a separate test and check a few possibilities